Makefile

```diff
@@ -20,9 +20,14 @@ run: ## Run via godep (without building)
 	go run *.go
 
 all: test build docker ## Test, build and docker image build
+	
+fmt: ## Run go fmt for all files in the project (excluding vendor)
+	go fmt $(shell go list ./... | grep -v /vendor/)
 
-test: ## Perform both unit and integration tests
-	go test -v -cover -tags "unit integration" $(TEST_PACKAGES)
```

Comment:
We had set it up so that you could have unit and integration test files in the same directory but yet still have the option to run only "unit" or only "integration" tests.

We want this option as typically unit tests can run a lot faster than integration tests so you want the option to run them individually based on a developers needs during dev.